curriculum/challenges/english/25-front-end-development/lab-one-time-password-generator/67c562286b29447da020d407.md

```diff
@@ -20,8 +20,8 @@ In this lab, you will generate a 6-digit OTP (One-Time Password) and display it
 
     - An `h1` element with the ID `otp-title` and text `"OTP Generator"`.
     - An `h2` element with the ID `otp-display` that either displays the message `"Click 'Generate OTP' to get a code"` or shows the generated OTP if one is available.
-    - A `p` element with the ID `otp-timer` that:
-        - Starts off empty.
+    - A `P` element with the ID otp-timer that:
```

Comment:
changes to this line are not appropriate, element names should stay lowercase, and the id should stay between backticks